From 6f39c8c4dd262cca89bd878f2e3148394e5a4178 Mon Sep 17 00:00:00 2001
From: Abhinaba Rakshit <quic_arakshit@quicinc.com>
Date: Wed, 14 May 2025 20:22:07 +0530
Subject: [PATCH 2/2] QCLINUX: Revert: FROMLIST: arm64: dts: qcom: qcs615-ride:
 Enable SDHC1 and SDHC2

This reverts commit 7b4a4e689d5a1745583e71f2d6d4edbf4cf6e43b
Since patch v5 is the updated upstream patch, this v1 patch
is reverted.

Signed-off-by: Abhinaba Rakshit <quic_arakshit@quicinc.com>
Upstream-Status: Inappropriate [Revert]
---
 arch/arm64/boot/dts/qcom/qcs615-ride.dts | 28 ------------------------
 1 file changed, 28 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/qcs615-ride.dts b/arch/arm64/boot/dts/qcom/qcs615-ride.dts
index fcb5b1e09b9d..ec32f3f5f8f8 100644
--- a/arch/arm64/boot/dts/qcom/qcs615-ride.dts
+++ b/arch/arm64/boot/dts/qcom/qcs615-ride.dts
@@ -462,34 +462,6 @@ &rpmhcc {
 	clocks = <&xo_board_clk>;
 };
 
-&sdhc_1 {
-	pinctrl-0 = <&sdc1_state_on>;
-	pinctrl-1 = <&sdc1_state_off>;
-	pinctrl-names = "default", "sleep";
-
-	vmmc-supply = <&vreg_l17a>;
-	vqmmc-supply = <&vreg_s4a>;
-
-	non-removable;
-	no-sd;
-	no-sdio;
-
-	status = "okay";
-};
-
-&sdhc_2 {
-	pinctrl-0 = <&sdc2_state_on>;
-	pinctrl-1 = <&sdc2_state_off>;
-	pinctrl-names = "default", "sleep";
-
-	cd-gpios = <&tlmm 99 GPIO_ACTIVE_LOW>;
-
-	vmmc-supply = <&vreg_l10a>;
-	vqmmc-supply = <&vreg_s4a>;
-
-	status = "okay";
-};
-
 &tlmm {
 	ethernet_defaults: ethernet-defaults-state {
 		mdc-pins {
-- 
2.34.1

